docs/helm/hydra.md

```diff
@@ -159,7 +163,7 @@ $ helm install -f ./path/to/hydra-config.yaml ory/hydra
 
 Additionally, the following extra settings are available:
 
-- `autoMigrate` (bool): If enabled, an `initContainer` running
+- `automigration` (bool): If enabled, an `initContainer` running
```

Comment:
The change was bigger, as the bool is now an object which takes holds several options. The simples in this context would be `automigration.enabled` which creates now a job instead.

```suggestion
- `automigration.enabled` (bool): If enabled, a `Job` running
```